### ISSUE TITLE
Multiple versions of wasm-bindgen when using `js-sys`.

### ISSUE THREAD

**Issue by tmccombs** [bazelbuild/rules_rust#834]
When I try to use the `js-sys`  crate, I end up with two different versions of wasm-bindgen. One from @rules_rust//wasm_bindgen/raze:wasm_bindgen and the other from the depedency of js-sys (using cargo-raze). 

What is the best way to work around this?

**Comment by UebelAndre** [comment]
Hi, when setting up your workspace, you'll want to define your own [rust_wasm_bindgen_toolchain](https://bazelbuild.github.io/rules_rust/rust_wasm_bindgen.html#rust_wasm_bindgen_toolchain) and make sure you set [`register_default_toolchain = False`](https://bazelbuild.github.io/rules_rust/rust_wasm_bindgen.html#rust_wasm_bindgen_repositories-register_default_toolchain) so that your toolchain with a matching `wasm_bindgen` rev is used.

Let me know if that fixes it for you 😄

**Comment by tmccombs** [comment]
Yeah, I got that to work, although it took a fair amount of trial and error.

**Comment by UebelAndre** [comment]
Yay! Glad you finally got it but sorry you had so much trouble 😞 

What was the trial and error you had to deal with? Also, I'm happy to review any changes to the API or documentation to avoid this in the future! Using the rules should be easy and fun, not a pain! 😅

**Comment by tmccombs** [comment]
It wasn't immediately obvious that I needed to define the toolchain in my BUILD.bazel but register the toolchain in the WORKSPACE. 

The `rust_bindgen_toolchain` in the example ws missing the `name` parameter, it should have been:

```
rust_bindgen_toolchain(
    name = "wasm_bindgen_toolchain_impl",
    bindgen = "//my/cargo_raze:cargo_bin_wasm_bindgen",
)
```


But my biggest problem was figuring out  how to build the wasm-bindgen cli with cargo raze.

It took me a few tries, and reading multiple pages on cargo raze to figure out that I needed to add both

```
[package.metadata.raze.crates.wasm-bindgen-cli.'*']
extra_aliased_targets = ["cargo_bin_wasm_bindgen"]
```

and

```
[package.metadata.raze.binary_deps]
wasm-bindgen-cli = "0.2.74"
```

to my Cargo.toml. 

And then it took me a while to figure out that I needed to add a package section to the parent Cargo.toml for the binary_deps to work (I eventually found a Github issue for cargo-raze for that).
